docs/build_node.md

```diff
@@ -381,7 +381,60 @@ This section assumes you've followed the steps of the [Setup](#setup) and [Runni
 
 ### Opening a Channel
 
-See the "Opening a Channel" guide.
+```java
+// <insert code to connect to peer via NioPeerHandler.connect(byte[] their_node_id, SocketAddress remote)>
+
+// Create the initial channel of 10000 sats with a push_msat of 1000 and make sure the result is successful.
+// The `42` here can be any value, and it is not used internally in LDK. It will be provided back to you
+// in the `Event.FundingGenerationReady` event as `user_channel_id`.
+byte[] peer_node_pubkey = <peer node pubkey bytes>;
+Result_NoneAPIErrorZ create_channel_result = channel_manager.create_channel(
+    peer_node_pubkey, 10000, 1000, 42, null);
+assert create_channel_result instanceof Result_NoneAPIErrorZ.Result_NoneAPIErrorZ_OK;
+
+// Ensure we immediately send a `create_channel` message to the counterparty.
+nio_peer_handler.check_events();
+
+// After the peer responds with an `accept_channel` message, an Event.FundingGenerationReady
+// event will be generated.
+
+// In the background event handler thread (see section above on "Handle...Generated Events"),
```

Comment:
Could you rename the event mentioned^ to "Handle LDK Events" and rename that here :grimacing: